src/bridge/AdminRoomHandler.ts

```diff
@@ -213,6 +221,47 @@ export class AdminRoomHandler {
         return new MatrixAction("notice", "Room plumbed.");
     }
 
+    private async handleUnlink(args: string[], sender: string, userPermission: string | undefined) {
+        if (userPermission !== 'admin') {
+            return new MatrixAction("notice", "You must be an admin to use this command");
+        }
+        const [matrixRoomId, serverDomain, ircChannel] = args;
+        const server = serverDomain && this.ircBridge.getServer(serverDomain);
+        if (!server) {
+            return new MatrixAction("notice", "The server provided is not configured on this bridge");
+        }
+        if (!ircChannel || !ircChannel.startsWith("#")) {
```

Comment:
Kinda surprised we don't have a static function to validate ircChannel names yet, that'd be handy rather than using the startsWith snippets over and over.